src/commands/publish/upload_form/project_assets.rs

```diff
@@ -58,7 +60,7 @@ impl ProjectAssets {
         self.script_name.to_string()
     }
 
-    pub fn script_path(&self) -> String {
-        self.script_path.to_string()
+    pub fn script_path(&self) -> PathBuf {
+        self.script_path.clone()
```

Comment:
do we even need this function anymore, or could we start using `self.script_path` directly and borrow it when we pass it into functions?

Reply:
yeah is interesting question; in other languages I have been in the habit of defining accessor methods for attributes, but I still feel shaky on the mutability constraints of Rust structs when they are made public.